alerts/cluster-monitoring-operator/PrometheusDuplicateTimestamps.md

```diff
@@ -56,10 +57,16 @@ Proceed with the following steps to fix the issue:
 1. Use the logs to guide you to the place where the conflicting targets are defined.
 2. Remove the duplicated targets or ensure that distinct targets are labeled uniquely.
 
-### The logs only revolve around the same scrape pool
+### Target is exposing duplicated samples for the same timestamp
+
+In this scenario, even if the samples have the same value for the same
+timestamp, they are considered duplicates.
 
-This might mean that the target exposes duplicated samples. In this scenario, even
-if the samples have the same value, they are considered duplicates.
+#### NOTE
+
+A regression introduced in OpenShift Container Platform 4.16.0 might cause the alert
+to fire when a target is exposing samples of the same series with different explicit
+timestamps (check [OCPBUGS-39179] to track the resolution).
```

Comment:
Just a small nitpick. It is not a dealbreaker for merge :slightly_smiling_face: 

```suggestion
timestamps (see [OCPBUGS-39179] to track the resolution).
```

Reply:
Otherwise LGTM

Reply:
I still have some force push quota :)

Reply:
I still have some force push quota :)